public/css/syntax.css

```diff
@@ -59,7 +59,7 @@
 .highlight .vg { color: #033 } /* Name.Variable.Global */
 .highlight .vi { color: #033 } /* Name.Variable.Instance */
 .highlight .il { color: #f60 } /* Literal.Number.Integer.Long */
-
+figure .highlight { margin-start: 0px; margin-end: 0px; }
```

Comment:
It took me 15 minutes to "figure" (har har har) this out, but this should read `figure.highlight` (no space)... I fixed it in master.

I have no idea why it worked locally, btw :-/

In related news, I tried and failed to get overflow text to scroll within the `<pre>` block but couldn't get it to work and gave up. I'm quite certain that it's possible, but whatever.